iohk/iohk-ops.hs

```diff
@@ -282,7 +283,14 @@ runTemplate o@Options{..} Template{..} = do
 runTemplate Options{..} _ = error "impossible"
 
 runSetRev :: Options -> Project -> Commit -> IO ()
-runSetRev o proj rev = do
+runSetRev o Nixpkgs newRev = do
+  oldGHSrc@Ops.GithubSource{..} <- Ops.readSource Ops.githubSource Nixpkgs
+  printf ("Changing nixpkgs commit from "%s%" to "%s%"\n") (fromCommit ghRev) (fromCommit newRev)
+  newSha256 <- incmd o "nix-prefetch-url" ["--unpack", fromURL $ Ops.nixpkgsRevTarball newRev]
```

Comment:
Due to https://github.com/input-output-hk/iohk-ops/pull/129 we should remove `--unpack` and test if the two PRs work together.